# common_mk_exts_mk_dep.patch
#
# Target:  common.mk (pass2 (linux gnu/musl, macos))
# Source:  tebako gem lib/tebako/packager/patch_buildsystem.rb PatchBuildsystem::COMMON_MK_PATCH
#
# Force rebuild of extinit.c when exts.mk changes (exts.mk changes when
# extensions are rebuilt to make them statically linked).
diff --git a/common.mk b/common.mk
index cddc6dc..bc0fdbd 100644
--- a/common.mk
+++ b/common.mk
@@ -335,7 +335,7 @@ build-ext: $(EXTS_MK)
 exts-note: $(EXTS_MK)
 	$(Q)$(MAKE) $(EXTS_NOTE)
 
-ext/extinit.c: $(srcdir)/template/extinit.c.tmpl $(PREP)
+ext/extinit.c: $(srcdir)/template/extinit.c.tmpl $(PREP) $(EXTS_MK)
 	$(MAKEDIRS) $(@D)
 	$(Q)$(MINIRUBY) $(tooldir)/generic_erb.rb -o $@ -c \
 	    $(srcdir)/template/extinit.c.tmpl $(EXTINITS)
